airbyte-integrations/connectors/source-amazon-seller-partner/integration_tests/configured_catalog_browse_tree_data.json

```diff
@@ -0,0 +1,136 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "GET_XML_BROWSE_TREE_DATA",
+        "json_schema": {
+          "title": "XML Browse Tree Data",
+          "description": "XML Browse Tree Data",
+          "type": "object",
+          "$schema": "http://json-schema.org/draft-07/schema#",
+          "properties": {
+            "browseNodeId": {
+              "type": ["null", "string"]
+            },
+            "browseNodeAttributes": {
+              "type": ["null", "object"],
+              "properties": {
+                "attribute": {
+                  "anyOf": [
+                    {
+                      "$ref": "#/definitions/attribute"
+                    },
+                    {
+                      "type": ["array"],
+                      "items": {
+                        "$ref": "#/definitions/attribute"
+                      }
+                    }
+                  ]
```

Comment:
It will make it easier for a destination to pick the right column type if you stick to arrays rather than `anyOf`, but this will require a small transformation in the parsing.

Reply:
@manescianera you removed the `anyOf` to make this property only an `array`, but I can't see any update in the parsing logic. Does it mean you figured the input data will always be an `array`?